.gitignore

```diff
@@ -0,0 +1,8 @@
+node_modules
+dist
+.DS_Store
+.idea
+.vscode
+yarn-error.log
+package-lock.json
```

Comment:
package-log and yarn-error together are a bit strange?

Reply:
removed